sql_generators/active_users_aggregates_v3/templates/mobile_query.sql

```diff
@@ -32,20 +32,39 @@ baseline AS (
     device_model,
     first_seen_date,
     submission_date = first_seen_date AS is_new_profile,
-    uri_count,
-    is_default_browser,
     CAST(NULL AS string) AS distribution_id,
     isp,
-    IF(
-      isp = 'BrowserStack',
-      CONCAT('{{ app_value }}', ' BrowserStack'),
-      '{{ app_value }}'
-    ) AS app_name
+    app_name
   FROM
-    `{{ project_id }}.{{ app_name }}.clients_last_seen_joined`
+    `{{ project_id }}.{{ app_name }}.baseline_clients_last_seen`
   WHERE
     submission_date = @submission_date
 ),
+metrics AS
+(
+  SELECT
+    submission_date,
+    client_id,
+    normalized_channel,
+    uri_count,
+    is_default_browser
+  FROM
+    `{{ project_id }}.{{ app_name }}.metrics_clients_last_seen`
+  WHERE
+    submission_date = DATE_ADD(@submission_date, INTERVAL 1 DAY) -- Metrics ping usually arrives 1 day after baseline ping.
+),
+unioned AS (
+  SELECT
+    baseline.*,
+    metrics.is_default_browser,
+    metrics.uri_count
+  FROM
+    baseline
+  LEFT JOIN
+    metrics
+    ON baseline.client_id = metrics.client_id
+    AND baseline.normalized_channel IS NOT DISTINCT FROM metrics.normalized_channel
```

Comment:
Is it possible that this join can result in double-counting like [we saw recently](https://mozilla.slack.com/archives/C05967995U5/p1713481147013339)?

Reply:
We could make [this check](https://github.com/mozilla/bigquery-etl/blob/12da7b0478b89a0fa4260ce1a3f8ce60ea307cf9/sql_generators/active_users_aggregates_v3/templates/mobile_checks.sql#L13) fail if the difference is duplicated.

Reply:
Maybe the checks on https://github.com/mozilla/bigquery-etl/pull/5324 are good enough for now, since they check for client_id uniqueness.

Reply:
It's a tricky problem. One possible solution is to build a test that compares the previous day's active_users_aggregates partition to data from the same period recalculated from this query. If the numbers are different by X% then there's possibly a data issue somewhere.

Either way I don't think it's a blocker, but it would be nice to have something in place to make sure we don't inflate DAU accidentally.

Reply:
Adding checks in [PR-5548](https://github.com/mozilla/bigquery-etl/pull/5548).